### PR TITLE
P1 redux

### PR DESCRIPTION
**Pull request by jingwanhsu**
not yet implement sendCreateIntentRequest

### CODE REVIEW

File: reducers/cyStyle.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import DefaultVisualStyle from '../css/DefaultVisualStyle'
 
 const cyStyle = (state = DefaultVisualStyle, action) => {
-  // TODO
   switch(action.type) {
+    case "UPDATE_INTENT_LABEL":
+      return state.map(s => {
+        if (s.selector == 'node') {
+          s.css.label = 'data(' + action.value + ')';
```
Comment (LaContra) [inline reducers/cyStyle.js]: Every s in the state is modified here. If you only want to modify some part of the values in s, you can create a new object and then modify it by using Object.assign(). Please refer to the  "ADD_INTENT" case in cyElements.js.

File: components/IntentProperty.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React, { PropTypes } from 'react'
+import $ from 'jquery'
```
Comment (LaContra) [inline components/IntentProperty.js:2]: I can't run the server locally. Did you modify package.json?

File: webpack.config.js [hunk 3]
```diff
@@ -6,10 +6,17 @@ module.exports = {
...
-			exclude: /node_modules/,
-			loader: "babel-loader"
-		}]
+		loaders: [
+			{
+				test: /\.js$/,
+				exclude: /node_modules/,
+				loader: "babel-loader"
+			},
+			{ test: /\.css$/, loader: "style-loader!css-loader" },
```
Comment (LaContra) [inline webpack.config.js:15]: There is a error msg "Module not found: Error: Cannot resolve module 'style-loader'" so I can't run the server.

File: components/IntentProperty.js (hunk 2 above)
Comment (jingwanhsu) [inline components/IntentProperty.js:2]: yes I did. plz install them

Comment (LaContra) [inline components/IntentProperty.js:2]: Well, that's weird. I didn't see the modification log for package.json.

Comment (LaContra) [inline components/IntentProperty.js:2]: I didn't see jquery and cytoscape in the package.json?

Comment (jingwanhsu) [inline components/IntentProperty.js:2]: ah I miss them.
jquery would be included with bootstrap, but it will be better to put it in package.json

**Pull request merged**